controllers/datanetwork_controller.go

```diff
@@ -407,8 +409,30 @@ func (r *DataNetworkReconciler) UpdateScopeConfig(instance *starlingxv1.DataNetw
 		return err
 	}
 
-	// Update status
+	// Update scope status
 	instance.Status.DeploymentScope = deploymentScope
+
+	// Set default value for StrategyRequired
+	if instance.Status.StrategyRequired == "" {
+		instance.Status.StrategyRequired = "not_required"
```

Comment:
Is it possible to write this as follows?
instance.Status.StrategyRequired = cloudManager.StrategyNotRequired

There are 6 instances of this "not_required" assignment.

Reply:
Yes, you are right. I should not hard-coded here. Updated.